go/cmd/vtcombo/main.go

```diff
@@ -36,6 +40,7 @@ var (
 	shardingColumnName = flag.String("sharding_column_name", "keyspace_id", "Specifies the column to use for sharding operations")
 	shardingColumnType = flag.String("sharding_column_type", "", "Specifies the type of the column to use for sharding operations")
 	vschema            = flag.String("vschema", "", "vschema file")
+	tabletTypesToWait  = flag.String("tablet_types_to_wait", "", "wait till connected for specified tablet types during Gateway initialization")
```

Comment:
For vtcombo, I would just hard-code master+replica+rdonly in the vtgate.Init call. We create one of each for each shard, and we want to wait for them all.